src/test/java/org/owasp/wrongsecrets/challenges/docker/Challenge4Test.java

```diff
@@ -26,6 +26,14 @@ void spoilerShouldRevealAnswer() {
     void rightAnswerShouldSolveChallenge() {
         var challenge = new Challenge4(scoreCard, "test");
 
+        Assertions.assertThat(challenge.solved("test")).isTrue();
```

Comment:
looks like this is an older MR coming back. This one is already fixed. Can you remove the Challenge4 related changes please? As these are already merged to master

Reply:
Yeah sorry was some git shenanigans. I created the MR too early

Reply:
no problem sir :). thanks!